content/docs/04-clusters/01-public-cloud/01-aws.md

```diff
@@ -946,6 +963,33 @@ To create an AWS cloud account using STS credentials follow the steps below:
 </Tabs.TabPane>
 </Tabs>
 
+## Create AWS US Gov Cloud Account
+
+ To deploy the AWS US Gov Clusters in the Palette Platform, having the AWS US Gov Cloud Account registered to the Palette Platform is a significant prerequisite. This section guides you on creating an AWS US Gov cloud account in the Palette console.

```

Comment:
Replace first sentence with a Prerequisite section that lists 'AWS US Gov Cloud Account registered to the Palette Platform' as the prerequisite.